test/test_operations.py

```diff
@@ -491,6 +491,10 @@ def runAtenTest(self, tensors, fn, device=None, rel_err=1e-2, abs_err=1e-5):
 
 class TestToXlaTensorArena(XlaTestCase):
 
+  @unittest.skipIf(
```

Comment:
let's revert all changes under `test_operations` if failure is related to opbyop, we can skip the opbyop on TPUCI for PJRT.

Reply:
Thanks, done~